### PR TITLE
Fix EMAIL_API_FROM_EMAIL default value

### PR DESCRIPTION
**Pull request by hectorgomezv**
Changes:
- Changes the `docker-compose.yml` default value for `EMAIL_API_FROM_EMAIL`.
- Changes the `test/e2e-setup.ts` default value for `EMAIL_API_FROM_EMAIL`.

Otherwise the following error would happen when running the container without providing a value via environment variable:
```
Error: Configuration is invalid: /EMAIL_API_FROM_EMAIL must match format "email"
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ services:
       ALERTS_PROVIDER_ACCOUNT: ${ALERTS_PROVIDER_ACCOUNT-example_account}
       ALERTS_PROVIDER_PROJECT: ${ALERTS_PROVIDER_PROJECT-example_project}
       EMAIL_API_APPLICATION_CODE: ${EMAIL_API_APPLICATION_CODE-example_application_code}
-      EMAIL_API_FROM_EMAIL: ${EMAIL_API_FROM_EMAIL-example_from_email}
+      EMAIL_API_FROM_EMAIL: ${EMAIL_API_FROM_EMAIL-ch@nge.me}
```
Comment (fmrsabino) [inline docker-compose.yml:42]: It's probably better to use `example.com` – https://www.iana.org/help/example-domains

Comment (hectorgomezv) [reply docker-compose.yml]: Thanks, I didn't know these domains are actually maintained by IANA 👍 Changed in [02d9ece](https://github.com/safe-global/safe-client-gateway/pull/942/commits/02d9ece0d4d8977a922c423eaccbe2d6a95e49d8)

File: test/e2e-setup.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ process.env.ALERTS_PROVIDER_API_KEY = 'fake-api-key';
 process.env.ALERTS_PROVIDER_ACCOUNT = 'fake-account';
 process.env.ALERTS_PROVIDER_PROJECT = 'fake-project';
 process.env.EMAIL_API_APPLICATION_CODE = 'fake-application-code';
-process.env.EMAIL_API_FROM_EMAIL = 'fake-from-email';
+process.env.EMAIL_API_FROM_EMAIL = 'ch@nge.me';
```
Comment (fmrsabino) [inline test/e2e-setup.ts:9]: We should change this one too

**Review by fmrsabino — CHANGES_REQUESTED** [review]

**Review by fmrsabino — APPROVED** [review]
🚀

**Pull request merged**